buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -43,7 +43,7 @@ conda list
 if [[ "$CONDA_SUBDIR" == "linux-32" || "$BITS32" == "yes" ]]; then
     conda create -n $CONDA_ENV -q -y ${EXTRA_CHANNELS} python=$PYTHON pip gitpython pyyaml
 else
-    conda create -n $CONDA_ENV -q -y ${EXTRA_CHANNELS} python=$PYTHON numpy=$NUMPY pip gitpython pyyaml
+    conda create -n $CONDA_ENV -q -y ${EXTRA_CHANNELS} python=$PYTHON numpy=$NUMPY pip gitpython pyyaml "setuptools<60"
```

Comment:
Didn't also add this to the linux-32 line above, it's implicit by there being no new packages created for linux-32 and it's scheduled for removal anyway.

I also checked that installing the other optional packages below didn't move the `setuptools` version through some transitive dependency, however this might have to be programmatically enforced at some point.